datasource/utils.go

```diff
@@ -4,6 +4,61 @@ package datasource
 
 import "strconv"
 
+var DivisionalAreas = [3]string{"Arts and Humanities", "Social Sciences", "Science & Engineering & Applied Science"}
+
+func getGenEdInfo(intAttributes []interface{}) []string {
+
+	// Type conversion, since we're dealing with an interface.
+	var attributes []map[string]interface {}
+	for _, item := range intAttributes {
+		attrMap, ok := item.(map[string]interface {}) 
+		if !ok {
+			panic("An item in courseAttributes is not of type map[string]interface")
+		}
```

Comment:
Can just make the type assertion directly without the `ok`, and it will panic automatically with a stack trace if it fails.